google/cloud/_testing.py

```diff
@@ -119,8 +119,6 @@ def __init__(self, *pages, **kwargs):
 
     def next(self):
         """Iterate to the next page."""
-        import six
-
-        return six.next(self._pages)
+        return next(self._pages)
```

Comment:
Since dropping Python 2, we could also drop the entire `next()` method and just use `__next__()`.

Reply:
9292b7b